examples/editor-core/overlays/constants.ts

```diff
@@ -0,0 +1,13 @@
+export const INITIAL_COORDS = {
+  lon: -8.545295433037069,
+  lat: 42.880392051799674
+};
+export const INITIAL_VIEW_STATE = {
+  longitude: INITIAL_COORDS.lon,
+  latitude: INITIAL_COORDS.lat,
+  zoom: 17,
+  pitch: 0,
+  bearing: 0
+};
+export const MAPBOX_ACCESS_TOKEN =
+  'pk.eyJ1IjoiZ2Vvcmdpb3MtdWJlciIsImEiOiJjanZidTZzczAwajMxNGVwOGZrd2E5NG90In0.gdsRu_UeU_uPi9IulBruXA';
```

Comment:
I suspect that we probably should not include a token in the example. For me, locally, the example renders correctly without an access token as well. @ibgreen any preference between:

- omitting the token from the example
- including a (possibly undefined) `import.meta.env.VITE_MAPBOX_ACCESS_TOKEN` 
- ... something else?

I suppose this would affect how the website is deployed later on.

Reply:
We should definitely avoid tokens in the repos. Is the choice of mapbox here important? Can we just migrate to token-less basemaps, such as the CARTO basemaps? If not let's use an env variable.

Reply:
I've downgraded react-map-gl to version 5 to avoid the need of using a token. Maybe this is needed in other examples too

Reply:
It was working without a token for me here, I'm not sure what's going on! In any case yes, we can change up the basemaps for any and all examples in a future PR. Thanks!